packages/angular/cli/commands/deploy-impl.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { ArchitectCommand, ArchitectCommandOptions } from '../models/architect-command';
+import { Arguments } from '../models/interface';
+import { Schema as DeployCommandSchema } from './deploy';
+
+export class DeployCommand extends ArchitectCommand<DeployCommandSchema> {
+  public readonly target = 'deploy';
+  public readonly error = `Cannot find "deploy" target for the specified project.
+
+You can find a package that implements deployment capabilities for your
+favorite platform on npm: https://www.npmjs.com/search?q=ng%20deploy
```

Comment:
We should provide a short list of 2-3 examples. Maybe @angular/fire, azure, etc?

Reply:
What would be the recommendation criteria? We don't have first party deployment builders.